GSoC-2025/application_template.md

```diff
@@ -10,7 +10,7 @@ Please use the following template to submit your application to the NIU GSoC 202
 The more closely you follow this template, the easier it will be for us to review your application! Please include clear headings for all the different sections.
 
 ## Project title
-Follow the following format for the proposal title: `<Package Name>:<Project Name> (<Your-Name>)`. 
+Follow the following format for the proposal title: `<Package Name>:<Project Name> (<Your-Name>).md`. 
```

Comment:
I was confused around why the "title" is an `.md` file? Should this be something like:
```suggestion
Follow the following format for the proposal title: `<Package Name>:<Project Name> (<Your-Name>)` and provide it in your pull request as a new markdown file of the same name, i.e.   `<Package Name>:<Project Name> (<Your-Name>).md`
```

The fact that it needs to be an `.md` file is mentioned in [the readme](https://github.com/neuroinformatics-unit/gsoc/blob/markdown/README.md#steps) but can't hurt to be extra clear.